WcaOnRails/app/views/competitions/_index_form.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="form-group">
```

Comment:
&nbsp; didn't work, as it would appear as the label text.

I had to make each button a separate div with a label_tag(' '); and add display: block to the buttons.